### ISSUE TITLE
Parameter for y axis font size and sparkline font size

### ISSUE THREAD

**Issue by nipunbatra** [ResidentMario/missingno#4]
Currently, these sizes are hardcoded. `ax0.set_yticklabels([1, df.shape[0]], fontsize=20)` and
 ```
ax1.annotate(max_completeness,
                 xy=(max_completeness, max_completeness_index),
                 xytext=(max_completeness + 2, max_completeness_index),
                 fontsize=14,
                 va='center',
                 ha='left')
```

I wonder if either of the two options could be provided:

1. Same font size is used everywhere (which is a parameter already)
2. Additional params are made available for tweaking these individual font sizes.

I would advocate 1 over 2 for simplicity. Would also be useful to allow usage of different fonts, like `serif`. Wonder if all this could be passed as `kwargs` to matplotlib.

**Comment by ResidentMario** [comment]
This would best be handled with some sort of `matrix_kwargs={}` and `sparkline_kwarags={}` argument.

**Comment by ResidentMario** [comment]
#3 addresses this issue by making the axes available for modification by returning them. I don't think it's useful to provide this as an additional parameter to the method on top of that--do you strongly think it would be, or do you think that it's fine to leave that to modifying the axes post-generation?

**Comment by ResidentMario** [comment]
Addressed prior to 3a5420e.
